server2/Arcadia.Assistant/Arcadia.Assistant.ExternalStorages.SharepointOnline/SharepointApiModels/SharepointListResponse.cs

```diff
@@ -0,0 +1,7 @@
+﻿namespace Arcadia.Assistant.ExternalStorages.SharepointOnline.SharepointApiModels
+{
+    public class SharepointListResponse
+    {
+        public string? ListItemEntityTypeFullName { get; set; }
```

Comment:
Specify `Data` serialization attributes here too, please, as it is a contract that is deserialized

Reply:
Not required for deserialization.